modules/sql/src/main/scala/EnumDef.scala

```diff
@@ -29,15 +29,20 @@ object EnumDef {
     implicit def caseDegrees [S <: Symbol] = at[(S, Degrees) ] { case (s, _) => "  val " + s.name + ": gem.math.Angle"}
     implicit def caseZoneId  [S <: Symbol] = at[(S, ZoneId)  ] { case (s, _) => "  val " + s.name + ": java.time.ZoneId"}
 
-    implicit def caseWavelengthNm[S <: Symbol] = at[(S, Wavelength.Nm ) ] { case (s, _) => "  val " + s.name + ": gem.math.Wavelength"}
-    implicit def caseWavelengthUm[S <: Symbol] = at[(S, Wavelength.Um ) ] { case (s, _) => "  val " + s.name + ": gem.math.Wavelength"}
+    implicit def caseWavelengthNm[S <: Symbol] = at[(S, Wavelength.Nm)] { case (s, _) => "  val " + s.name + ": gem.math.Wavelength"}
+    implicit def caseWavelengthUm[S <: Symbol] = at[(S, Wavelength.Um)] { case (s, _) => "  val " + s.name + ": gem.math.Wavelength"}
+
+    implicit def caseSlitWidthNm [S <: Symbol] = at[(S, SlitWidth.Nm) ] { case (s, _) => "  val " + s.name + ": gem.math.Wavelength"}
+    implicit def caseSlitCenterNm[S <: Symbol] = at[(S, SlitCenter.Nm)] { case (s, _) => "  val " + s.name + ": gem.math.Wavelength"}
```

Comment:
It doesn't make much sense to talk about slit widths in terms of a wavelength. They are measurement of length though, just like a wavelength.  What we've been doing is punting in cases like these and just storing unit-less primitives for now.

Reply:
I can see wavelength doesn't make sense for slit but here Wavelength was intended as a placeholder since it was already quite similar, but I'll use `Int` instead as you mentioned in another comment.